apps/admin-gui/src/app/shared/components/create-service-member-dialog/create-service-member-dialog.component.ts

```diff
@@ -152,25 +130,8 @@ export class CreateServiceMemberDialogComponent implements OnInit, AfterViewInit
     });
   }
 
-  existingLoginValidator(): AsyncValidatorFn {
-    return (control: AbstractControl): Observable<ValidationErrors> => {
-      let namespace: string = (
-        this.secondFormGroup.get('namespaceCtrl').value as string
-      ).toLowerCase();
-      namespace = namespace === 'not selected' ? 'mu' : namespace;
-      return control.valueChanges.pipe(
-        debounceTime(500),
-        take(1),
-        switchMap(() =>
-          this.usersManagerService
-            .isLoginAvailable(namespace, control.value as string)
-            .pipe(map((res) => (res ? null : { loginExists: true })))
-        )
-      );
-    };
-  }
-
   onCreate(): void {
+    this.cd.detach();
```

Comment:
Not sure why we detach here. When detached the component will not update with changes e.g. loading indicator is not shown because of this.